src/js/components/InfiniteScroll/stories/ItemHeights.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+import isChromatic from 'chromatic/isChromatic';
+import { grommet } from 'grommet/themes';
+
+import { Grommet, Box, InfiniteScroll, Text } from 'grommet';
+
+// import { allItems } from './Basics';
+const allItems = Array(240)
+  .fill()
+  .map((_, i) => i + 1);
+
+const ItemHeights = props => {
+  const [items, setItems] = useState(allItems.slice(0, 50));
+
+  const onMore = () => {
+    setTimeout(() => {
+      setItems(allItems.slice(0, items.length + 50));
+    }, 1000);
+  };
+
+  return (
+    <Grommet theme={grommet}>
+      <Box>
+        <InfiniteScroll items={items} onMore={onMore} {...props}>
+          {item => (
+            <Box
+              key={item}
+              height={item <= 25 ? 'xsmall' : 'xxsmall'}
+              pad="medium"
+              border={{ side: 'bottom' }}
+              align="center"
+            >
+              <Text>item {item}</Text>
+            </Box>
+          )}
+        </InfiniteScroll>
+      </Box>
+    </Grommet>
+  );
+};
+
+if (!isChromatic()) {
+  storiesOf('InfiniteScroll', module)
+    .add('Variable Item Height', () => <ItemHeights />)
+    .add('Variable Item Height w/replace', () => <ItemHeights replace />);
```

Comment:
I believe the story naming convention is moving towards sentence casing.

Also, these stories have been useful from a QA standpoint, but I don't know if they add much for end users. I am bit torn on the balance for need for behavioral QA vs. instructive benefit.

Reply:
I dont think they add anything for end users. I'm ok with removing these, but I do agree that it does help with manual testing.